# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../udev/hotfix-install.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From: Harald Hoyer <harald@redhat.com>
Date: Tue, 11 Sep 2007 15:20:54 +0000 (+0200)
Subject: only install *.rules
X-Git-Url: http://git.kernel.org/?p=linux%2Fhotplug%2Fudev.git;a=commitdiff_plain;h=554074c9ca2f1b8876176887e4656d06b870f2cd

only install *.rules
---

diff --git a/Makefile b/Makefile
index 39ffc7d..92ca846 100644
--- a/Makefile
+++ b/Makefile
@@ -230,9 +230,9 @@ install-config:
 	@ if [ ! -r $(DESTDIR)$(configdir)/udev.conf ]; then \
 		$(INSTALL_DATA) etc/udev/udev.conf $(DESTDIR)$(configdir); \
 	fi
-	@ for i in $(shell ls -1 etc/udev/rules.d); do \
-		if [ ! -r $(DESTDIR)$(configdir)/rules.d/$$i ]; then \
-			$(INSTALL_DATA) etc/udev/rules.d/$$i $(DESTDIR)$(configdir)/rules.d; \
+	@ for i in etc/udev/rules.d/*.rules; do \
+		if [ ! -r $(DESTDIR)$(configdir)/rules.d/$$(basename $$i) ]; then \
+			$(INSTALL_DATA) $$i $(DESTDIR)$(configdir)/rules.d; \
 		fi \
 	done
 	@ extras="$(EXTRAS)"; for target in $$extras; do \
